defi/lending_platform/setup_and_run.sh

```diff
@@ -0,0 +1,90 @@
+resim reset
```

Comment:
I suggest looking into revup (https://github.com/RadGuild/revup). It allows you to do things like that but in a more readable way.

Reply:
Looks interesting. Will take a look in the near future!